DOC-2129: Create Access Control and Data Access Control landing pages (#2489)
RedisLabs/redislabs-docs: content/rc/security/access-control/data-access-control/_index.md

         /rc/security/database-security/passwords-users-roles.md
---

## [Default user]({{<relref "/rc/security/access-control/data-access-control/default-user">}})

When you create a Redis Cloud database, your database is given a randomly generated password called the **Default user password**. Learn how to [change the default user password]({{<relref "/rc/security/access-control/data-access-control/default-user#change-password">}}) or [turn off access]({{<relref "/rc/security/access-control/data-access-control/default-user#turn-off-default-user">}}) using the default user password.

## [Role-based access control]({{<relref "/rc/security/access-control/data-access-control/role-based-access-control.md">}})

> Role-based access control (RBAC) lets you define roles with specific sets of permissions to access your databases. You can then assign database users to these roles to provide appropriate levels of database access.

Reviewer: I think we might be able to simplify the phrasing here. I added a suggestions to give you an idea of how we could cut down on the word count here. You might have a better way to phrase it. 
```suggestion
Role-based access control (RBAC) lets you assign users different levels of access based off roles you define. 
```

Author: ```suggestion
With Role-based access control (RBAC), you create roles and assign your users to those roles to grants different levels of access to the database.
```
